src/Storefront/Page/Address/Listing/AddressListingPageLoader.php

```diff
@@ -101,15 +100,15 @@ private function getSalutations(SalesChannelContext $context): SalutationCollect
     private function getCountries(SalesChannelContext $salesChannelContext): CountryCollection
     {
         $criteria = (new Criteria())
-            ->addFilter(new EqualsFilter('country.active', true))
```

Comment:
same

Reply:
Yes, that's already default of the `CountryRoute`